.github/workflows/requirements/style/requirements.txt

```diff
@@ -2,6 +2,6 @@ black==24.10.0
 clingo==5.7.1
 flake8==7.1.1
 isort==5.13.2
-mypy==1.11.2
+mypy==1.8.0
```

Comment:
No need to fall-back on this, since: https://github.com/python/mypy/blob/3b467509ee29b8f274c035d78a1c241a781eb311/setup.py#L11-L13